docs/reference/cli-commands/cli-commands.md

```diff
@@ -4,11 +4,31 @@
 :titlesonly:
 :maxdepth: 1
 
-help <help>
-logs <logs>
-replan <replan>
 run <run>
+help <help>
```

Comment:
Not a comment on this line, but can we add a one-paragraph into to this page? How `pebble` uses subcommands as `pebble foo`, and that this is the list of all commands, something like that?